ophyd/areadetector/trigger_mixins.py

```diff
@@ -30,11 +30,15 @@ class TriggerBase(BlueskyInterface):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # settings
-        self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop.
-                                (self.cam.image_mode, 1),  # 'Multiple' mode
-                                ])
+        # careful here: quadEM devices have areadetector components but,
+        # they have no 'cam' plugin. See QuadEM initializer.
+        if hasattr(self, 'cam'):
+            self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop
+                                    (self.cam.image_mode, 1),  # 'Multiple' mode
+                                    ])
+            self._acquisition_signal = self.cam.acquire
```

Comment:
I think adding a `cam_attr` kwarg to `TriggerBase` itself is fine (no need for an extra layer IMO). I would prefer that it have the default value `'cam'` for back-compat. Since `TriggerBase` is scoped in the areadetector module I think it's OK that it makes some assumptions about how area detector attributes are named by default. But making it configurable is a good idea.